### ISSUE TITLE
BCI Competition IV 2b results

### ISSUE THREAD

**Issue by sansiro77** [comojin1994/m-shallowconvnet#1]
I guess the accuracy for each subject is from training on the dataset of 3 T files and evaluating the dataset of 2 E files.
How about cross-subject results, or specifically, leave-one-out-cross-validation results?
And by the way, do you know the related SOTA results? Thank you.

**Comment by comojin1994** [comment]
Hi @sansiro77,

Yes, I evaluated the performance of BCI competition 4 2b using two evaluation files which were the original test data training with three training files.

However, I didn't evaluate the subject independent tasks. Also, I didn't research the SOTA results in subject independent tasks.

**Comment by sansiro77** [comment]
I also wonder why you choose [`tmin, tmax = 0., 3.`](https://github.com/comojin1994/m-shallowconvnet/blob/55fa018167ee3935379cf86aad395d91d3f5578c/dataloader/bci_compet.py#L197).
I get the dataset from MOABB, so the data format may be a little different.
However, according to the original [description file](https://www.bbci.de/competition/iv/desc_2b.pdf), the imagery period is from 4s to 7s without feedback or from 3.5s to 7.5s with feedback.
Did I miss something?

**Comment by comojin1994** [comment]
@sansiro77 

The format of MOABB and original data is different.

In my case, `tmin, tmax = 0., 3.` indicates the 3 to 6 sec..
The reason why I chose an interval like this is that many previous studies commonly used the data containing cue time.
I think the features of the motor imagery were also contained before the imagery period like MRCP.
For this reason, I contained the cue time data.

The reason why set the tmax as 3 is that I optimized the interval, which showed the best performance.

However, in my custom data, I verified that this model also worked well when conducted only in the imagery period.
